.github/workflows/link-health-check.yml

```diff
@@ -0,0 +1,40 @@
+---
+# This workflow uses lychee to check all link in a base path periodically
+name: URL Link Health Checks
+
+on:
+  schedule:
```

Comment:
There is already a link check every week, although I think I'm the only one getting (and ignoring!) these reports